runtime/kernel/src/sys/win/dsys_interface.cpp

```diff
@@ -771,7 +771,8 @@ LTRESULT dsi_DoErrorMessage(const char *pMessage) {
 
 void dsi_MessageBox(const char *pMessage, const char *pTitle) {
     int i;
-    i = MessageBox(g_ClientGlob.m_hMainWnd, pMessage, pTitle, MB_OK);
+    i = SDL_ShowSimpleMessageBox(SDL_MESSAGEBOX_ERROR,
```

Comment:
good to have a more generic SDL version. Thank you.
I'll plan on moving this to the linux dsys.